extensions/rich_text_components/Collapsible/directives/collapsible.directive.html

```diff
@@ -11,3 +11,50 @@
     </div>
   </div>
 </uib-accordion>
+
+<!-- TODO(ankita240796): This styling is required since ui-bootstrap does not work correctly with bootstrap 4. We need to switch from ui-bootstrap to ng-bootstrap once we have upgraded the collapsible component to Angular 8.-->
+<style>
+  .panel {
```

Comment:
Should these be scoped to just this collapsible directive (by adding a suitable CSS prefix)? Otherwise, I am concerned that this will affect other .panel classes throughout the app which may be added in the future.

Ditto for all the CSS rules below.

Reply:
Done.